### PR TITLE
Issue/26 Implement roles

### PR DESCRIPTION
**Pull request by cerbin1**

### CODE REVIEW

File: src/main/java/com/project/server/startap/user/entity/Role.java [hunk 1]
```diff
@@ -5,10 +5,9 @@
 import javax.persistence.*;
 
 @Data
-@Entity(name = "roles")
+@Entity(name="roles")
 public class Role {
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
Comment (batormichal) [inline src/main/java/com/project/server/startap/user/entity/Role.java:11]: dlaczego?

File: src/main/java/com/project/server/startap/user/repository/RoleConstant.java [hunk 2]
```diff
@@ -18,25 +19,32 @@ public RoleConstant(RoleRepository roleRepository) {
...
             Role userRole = RoleConstant.getRoleByName("ROLE_USER");
             roles.add(userRole);
         } else {
-            strRoles.forEach(role -> {
+            roleAsString.forEach(role -> {
                 Role newRole;
                 switch (role) {
-                    case "admin":
-                        newRole = RoleConstant.getRoleByName("ROLE_ADMIN");
+                    case "guest":
```
Comment (batormichal) [inline src/main/java/com/project/server/startap/user/repository/RoleConstant.java:36]: guest to nie rola

File: src/main/resources/data.sql [hunk 3]
```diff
@@ -5,9 +5,11 @@ VALUES (102, 'example');
 INSERT INTO example
 VALUES (103, 'example');
 
-INSERT INTO roles(name) VALUES ('ROLE_USER');
-INSERT INTO roles(name) VALUES ('ROLE_MODERATOR');
-INSERT INTO roles(name) VALUES ('ROLE_ADMIN');
+INSERT INTO roles(id, name) VALUES (0, 'ROLE_GUEST');
```
Comment (batormichal) [inline src/main/resources/data.sql:8]: guest

Comment (cerbin1) [reply src/main/resources/data.sql]: fixed

File: src/main/java/com/project/server/startap/user/repository/RoleConstant.java (hunk 2 above)
Comment (cerbin1) [reply src/main/java/com/project/server/startap/user/repository/RoleConstant.java]: fixed

File: src/main/java/com/project/server/startap/user/entity/Role.java (hunk 1 above)
Comment (cerbin1) [reply src/main/java/com/project/server/startap/user/entity/Role.java]: fixed

**Comment by cerbin1** [comment]
@batormichal  Commit z poprawkami https://github.com/batormichal/startap/pull/28/commits/8b60119168c802ad6a96a925a2ef670c22340af0

**Pull request merged**